pkg/scheduler/scheduler.go

```diff
@@ -322,6 +357,26 @@ type entry struct {
 	preemptionTargets     []*preemption.Target
 }
 
+// NetUsage returns how much capacity this entry will require from the ClusterQueue/Cohort.
+// When a workload is preempting, it subtracts the preempted resources from the resources
+// required, as the remaining quota is all we need from the CQ/Cohort.
+func (e *entry) NetUsage() resources.FlavorResourceQuantitiesFlat {
+	if e.assignment.RepresentativeMode() == flavorassigner.Fit {
+		return e.assignment.Usage
+	}
+
+	usage := maps.Clone(e.assignment.Usage)
+	for target := range e.preemptionTargets {
+		for fr, v := range e.preemptionTargets[target].WorkloadInfo.FlavorResourceUsage() {
+			if _, hasFlavor := usage[fr]; !hasFlavor {
```

Comment:
```suggestion
			if _, uses := usage[fr]; !uses {
```

It cannot be has**Flavor**, because it's the flavor and resource.